helper/src/components/portalnav.js

```diff
@@ -295,6 +301,7 @@ export default function PortalNav({ config }) {
       }
     }
 
+    setLastAIUpdated({tab: tab, field: field})
```

Comment:
Hey Sam,  just one optimisation, you can put this in the body of your new if statement no?  so if the tab/field hasn't changed, you dont need to update the state?